pennylane/_qubit_device.py

```diff
@@ -849,7 +849,7 @@ def access_state(self, wires=None):
 
         return state
 
-    def generate_samples(self):
+    def generate_samples(self, shots: int = None):
```

Comment:
We allow overriding `shots` to take in single sample in `DefaultMixed::_apply_operation_midmeasuremp`.